src/promptflow-core/promptflow/core/_serving/app_base.py

```diff
@@ -55,7 +56,15 @@ def init_app(self, **kwargs):
         self.sample = get_sample_json(self.project_path, logger)
 
         self.init = kwargs.get("init", {})
-        logger.info("Init params: " + str(self.init))
+        try:
+            init_params = os.environ.get(PROMPTFLOW_FLOW_INIT_CONFIG, "{}")
+            init_dict: dict = json.loads(init_params)
+            init_dict.update(self.init)
+            self.init = init_dict
+        except Exception as e:
+            logger.error("Failed to retrieve init params from environment variable PROMPTFLOW_FLOW_INIT_CONFIG: ", e)
```

Comment:
IF init data format is not correct, deployment should fail instead of just logging the exception

Reply:
ok, I will cancel try catch.